README.md

```diff
@@ -95,9 +95,9 @@ Example: `<acrolinx.version>2.2.1</acrolinx.version>`
 
 ## Requirements
 
-AEM Cloud SDK: `2023.5.11983.20230511T173830Z-230200` or above <https://central.sonatype.com/artifact/com.adobe.aem/aem-sdk-api/2023.5.11983.20230511T173830Z-230200>
+AEM Cloud SDK: `2023.3.11382.20230315T073850Z-230200` or above <https://central.sonatype.com/artifact/com.adobe.aem/aem-sdk-api/2023.3.11382.20230315T073850Z-230200>
```

Comment:
question: Why do we lower the version? Was there a conflict with the higher one?

@abhijeetnarvekar

Reply:
Yes, you can find a detailed video about it in the issue JIRA https://acrolinx.atlassian.net/browse/DEV-34843?focusedCommentId=338386